packages/blast-utilities/config-utils.js

```diff
@@ -12,20 +12,36 @@ function getConfig() {
   return config
 }
 
-function getNetworkUrl() {
+function getNetwork(network) {
   const { config } = getConfig()
 
-  if (!config.networkUrl) {
-    throw new BlastError('Missing networkUrl in the config file.')
+  if (network) {
+  // network is passed - return it
+    if (!config.networks || !config.networks[network]) {
+      throw new BlastError(`Missing network: [${network}] from the config file.`)
+    }
+    return config.networks[network]
   }
-  return config.networkUrl
+  // network is not passed
+  // if default network is set - return it
+  if (config.defaultNetwork) {
+    if (!config.networks || !config.networks[config.defaultNetwork]) {
+      throw new BlastError(`Missing default network: [${config.defaultNetwork}] from the config file.`)
+    }
+    return config.networks[config.defaultNetwork]
+  }
+  // if default network is missing - return the local network
+  if (!config.localNetwork) {
```

Comment:
the local network shound't be in the config file of the user, but rather in the config file of the blast. The user must not be able to chagne the config file, because we spawn the node

Reply:
the proposed solution in the previous comment will also fix this